nabs/tests/test_utils.py

```diff
@@ -0,0 +1,48 @@
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+from pcdsdevices.device import GroupDevice
+
+from nabs.utils import post_ophyds_to_elog
+
+
+class StatusDevice(Device):
+    """ simulate a device with a status method """
+    def status(self):
```

Comment:
For reference: https://github.com/pcdshub/pcdsdevices/blob/aa724f54473fff8873c29bb40d6634ff3157390a/pcdsdevices/interface.py#L265-L270